### PR TITLE
Update s390 docker image for JDK8 to use Adopt-OpenJ9

### PR DESCRIPTION
**Pull request by babsingh**
- fixes https://github.com/eclipse/openj9/issues/1526
- [skip ci]

Signed-off-by: Babneet Singh <sbabneet@ca.ibm.com>

### CODE REVIEW

File: buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile [hunk 1]
```diff
@@ -83,4 +79,13 @@ RUN cd /root \
   && tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2 \
   && rm -f freemarker.tgz
 
+RUN cd /root \
```
Comment (DanHeidinga) [inline buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: Can you add a comment describing this?

File: buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile [hunk 2]
```diff
@@ -83,4 +79,13 @@ RUN cd /root \
...
+  && curl -OLJ https://api.adoptopenjdk.net/openjdk8-openj9/releases/s390x_linux/latest/binary \
```
Comment (DanHeidinga) [inline buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: Is there a reason to introduce `curl` when the image already has `wget`?

File: buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile (hunk 1 above)
Comment (babsingh) [reply buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: added comment.

File: buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile (hunk 2 above)
Comment (babsingh) [reply buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile]: no. we can also use `wget`. removed `curl`. changed download link to use `wget`. will use `wget` in #1514 and #1541 as well.

**Comment by DanHeidinga** [comment]
@babsingh Have you tested building this image on a s390 machine and building the VM in it?

**Comment by babsingh** [comment]
@jdekonin has access to an Ubuntu s390 machine where Docker can be run with root permissions. Using the instructions provided below, Joe successfully built the image on a s390 machine and built the VM in it.

```
    Build docker: docker build -f buildenv/docker/jdk8/s390x/ubuntu16/Dockerfile -t=openj9 .
    Run docker: docker run -i -t openj9:latest /bin/bash
    Test boot jdk: java -version
    Build Java 8
        git clone https://github.com/ibmruntimes/openj9-openjdk-jdk8.git
        cd openj9-openjdk-jdk8
        bash ./get_source.sh
        bash ./configure --with-freemarker-jar=/root/freemarker.jar
        make images
    Test Java 8: build/<platform>/images/jdk/bin/java -version
```

**Comment by babsingh** [comment]
@DanHeidinga these changes have been approved. can these changes be merged?

**Comment by DanHeidinga** [comment]
Merged.  Thanks @babsingh I had missed the comment showing the successful creation and build with the 390 image.

**Pull request merged**